assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -24,7 +24,22 @@ isolated_cores={{.IsolatedCpus}}
 {{end}}
 
 not_isolated_cores_expanded=${f:cpulist_invert:${isolated_cores_expanded}}
-automatic_pstate=${f:intel_recommended_pstate}
+
+# Check for x86 tuning
+{{if ${f:lscpu_check:"^Architecture:\s*(.*)$"} == "x86_64"}}
```

Comment:
I am not sure this is correct, is tuned evaluating the `{{if` or golang in the operator? I suspect the latter. And in that case it will not work.

Reply:
Also the includes need to happen in the [main] section. An example is on line 8

Reply:
I am not sure the order of the evaluation either but I am working on getting a lab up and running to try it and see what actually happens

Reply:
After testing I think you are correct, it seems that the curly braces are invalid characters in the [main] section

I have 2 more ideas for how this may be workable:
1. Include all the platform tuning in the main file and use conditions inside the individual files to no-op non matching platforms
2. Invert the include order so that we actually apply a specific platform profile (intel, amd, aarch64) and each of those import the existing openshift-node-performance profile with all the shared configuration

I am currently leaning more into 2 but I am not sure how the profile selection will work

Reply:
I've updated the performance profiles for idea 2, however this will require additional work on selecting the profile